lisp_ease/src/queries/designators.lisp

```diff
@@ -0,0 +1,144 @@
+(in-package :le)
+
+;; --- DESIG FACT GROUP --------------------------------------------------------------------
+;; the fact groups were taken from Gayane Kazhoyan's "mobile-pick-and-place-plans" and were adapted to the needs of this thesis. (link to original: https://github.com/cram2/cram/blob/master/cram_common/cram_mobile_pick_place_plans/src/pick-place-designators.lisp)
+
+(cram-prolog:def-fact-group ba-pnp-object-knowledge (object-rotationally-symmetric orientation-matters object-type-grasp)
+
+  (cram-prolog:<- (orientation-matters ?object-type)
+    (member ?object-type (:ba-fork)))
+  (cram-prolog:<- (object-type-grasp :ba-fork :human-grasp))
+  (cram-prolog:<- (object-type-grasp :ba-milk :human-grasp))
+  (cram-prolog:<- (object-type-grasp :ba-cup :human-grasp))
+  (cram-prolog:<- (object-type-grasp :ba-muesli :human-grasp))
+  (cram-prolog:<- (object-type-grasp :ba-bowl :human-grasp)))
+
+(cram-prolog:def-fact-group pick-and-place-plans (desig:action-grounding)
```

Comment:
Please check if there are any changes from cram_common/src/cram_pick_place_designators.lisp.
Make sure there are none, and delete this code.